README.md

```diff
@@ -1,9 +1,10 @@
 # STEP 2020 Capstone project
 
-Link to the Website: https://step2020-team36-capstone.appspot.com/
+Link to the Website (personal deployment): http://engage-step.appspot.com/
```

Comment:
[Optional] It probably makes more sense just to just use engage.appspot.com if it's available.  It's shorter and once you move the project outside Google you may want to continue working on it outside the STEP project.